src/status_im/contexts/wallet/swap/swap_proposal/view.cljs

```diff
@@ -12,4 +12,8 @@
      [quo/button
       {:on-press #(rf/dispatch [:show-bottom-sheet
                                 {:content slippage-settings/view}])}
-      (str "Edit Slippage: " max-slippage "%")]]))
+      (str "Edit Slippage: " max-slippage "%")]
+     [quo/button
+      {:on-press #(rf/dispatch [:navigate-to-within-stack
+                                [:screen/wallet.swap-confirmation :screen/wallet.swap-propasal]])}
+      "Swap confirmation"]]))
```

Comment:
Temporary button, will be removed

Reply:
removed when? 🤔  before merging?

Reply:
Not before merging. All the changes are under the swap feature flag. Currently, this button is added to a dummy screen to make parallel development possible. We will remove it at a later stage when we implement the actual UI for this screen.